lang/lang-check.py

```diff
@@ -230,6 +230,14 @@ def check_translation(entry, msgids, is_pot, no_warning, no_suggest, warn_empty,
                                 wrapped_source, wrapped_translation,
                                 rows, cols)
 
+    # Check for translation length shorter
```

Comment:
This looks good, but does it really help?
We have also "Short translation" which warns if the translation if less than half the size of the original.

Reply:
Yes it helps me to find way too many rows defined for translations.
If someone would use all available space (defined bye c= r=) for the translations we would get ~18816bytes to squeeze into the flash/xflash. But this is of course not possible as the xflash layout just allows 15616 bytes to be used for 2nd language.

I hope to get the rows as small/needed as possible. Avoiding novels being written on 20x4 LCD screen.

Reply:
Got it